orfs

```diff
@@ -2,7 +2,11 @@
 set -ex
 WORKSPACE="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-export ORFS=~/OpenROAD-flow-scripts
```

Comment:
Use the `. env.sh` variable convention, FLOW_HOME, no need to introduce a the ORFS variable:

https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/470955ddac63b2b571e6a83e7da8734cd7822397/env.sh#L25

Reply:
`ORFS` removed, now only `FLOW_HOME` is used